specification/specs/common.ts

```diff
@@ -99,8 +101,8 @@ type ScrollIds = string // TODO: array of ScrollIds
 type CategoryId = string
 type ActionIds = string // TODO: check if this should be an array of ActionId
 
-type Id = string
-type Ids = string | string[]
+type Id = string | long
```

Comment:
This does not feel right, can you point me to the yaml test that is failing?

Reply:
the `ids_query` accepts both

Reply:
Then we should add this edge case only there, every other API needs a string and never a number :)

Reply:
I actually would vote to not have a union for `Id` it's a string everywhere else

Reply:
updated, it was in two places (pinned_query and LikeDocument).